dumb-jump.el

```diff
@@ -1314,26 +1394,58 @@ Ffrom the ROOT project CONFIG-FILE."
                           " --search-zip"
                         "")
                       (s-join "" (--map (format " --%s" it) agtypes))))
-         (exclude-args (dumb-jump-arg-joiner "--ignore-dir" (--map (s-replace proj "" it) exclude-paths)))
-         (regex-args (format "\"%s\"" (s-join "|" filled-regexes))))
+         (exclude-args (dumb-jump-arg-joiner
+                        "--ignore-dir" (--map (shell-quote-argument (s-replace proj "" it)) exclude-paths)))
+         (regex-args (shell-quote-argument (s-join "|" filled-regexes))))
+    (if (= (length regexes) 0)
+        ""
+        (dumb-jump-concat-command cmd exclude-args regex-args proj))))
+
+(defun dumb-jump-generate-rg-command (look-for cur-file proj regexes lang exclude-paths)
+  "Generate the rg response based on the needle LOOK-FOR in the directory PROJ."
+  (let* ((filled-regexes (dumb-jump-populate-regexes look-for regexes 'rg))
+         (agtypes (dumb-jump-get-ag-type-by-language lang))
+         (cmd (concat dumb-jump-rg-cmd
+                      " --color never --no-heading --line-number"
+                      (s-join "" (--map (format " --type %s" it) agtypes))))
```

Comment:
Looks like `agtypes` can't be used with `rg` since some don't line up. For instance, `ag` uses `python` and `rg` uses `py` for `--type`, which basically means python won't work if you have `rg` installed.